### DISCUSSION TITLE
Modify checkbox/label placement

### DISCUSSION THREAD

**Discussion by jpsteil** [emmett-framework/emmett#426] [Q&A]
In my forms I like to have my checkbox label appear inline, after the checkbox.

With the default form formatting, the checkbox is displayed just like any other field, with the label above it.

I've looked through the form code but don't see how I could alter this get the results I'm looking for.

Am I missing something obvious?

**Comment by gi0baro** [comment]
@jpsteil the HTML structure of the forms generated by Emmett is covered by the `FormStyle` class, which – I'm really sorry about this – is not yet documented.

My question for you would be: since the HTML elements generated by Emmett have classes and ids, won't you be able to do this just with a CSS rule?
